kernel/src/arch/x86_64/mod.rs

```diff
@@ -475,6 +506,54 @@ fn identify_numa_affinity(
     );
 }
 
+/// Map the persistent memory addresses to the vspace.
+fn map_physical_persistent_memory() {
```

Comment:
is there a reason we can't do this as part of the bootloader?

Reply:
As discussed earlier, UEFI does not handle PMem related tables. Bespin gets PMem related information through ACPI tables (using topology crate).

Reply:
Yea strange. FWIW I just tried to upgrade our OVMF to the latest nightly and the pmem regions still don't show up when we query the uef memory map in the bootloader :/

Reply:
We can keep the TODO comment. 

However, IIRC, SNIA intentionally didn't put the logic in UEFI but ACPI. They didn't want to change the firmware again and again for each change, and eventually, they also wanted to move towards a plug-and-play model for CXL(PCIe 5.0).